.github/workflows/notification.yml

```
@@ -0,0 +1,89 @@
+name: Send Notification
+
+on:
+  issues:
+    types: [opened]
+  pull_request:
+    types: [opened]
+
+jobs:
+  job1:
```

Shall we name the Job :)
```suggestion
  job1:
```

---

Resolved by [554ac81](https://github.com/asgardeo/asgardeo-auth-js-sdk/pull/176/commits/554ac81a0246fe6c982ae4e66bcc83344546a7de)